ta/arch/arm32/user_ta_header.c

```diff
@@ -35,6 +35,15 @@
 #define TA_TRACE_LEVEL_DEFAULT CFG_TRACE_LEVEL
```

Comment:
Agreed, I also prefer the original way if you want to keep TA_TRACE_LEVEL_DEFAULT. My question was more: where is TA_TRACE_LEVEL_DEFAULT used? Is it documented somewhere as an interface that we don't want to break? What does it do that setting CFG_TRACE_LEVEL would not allow? CFG_TRACE_LEVEL could be the only external interface to the trace_level variable.